### PR TITLE
Added (failing) test cases for FHIR -> OHDSI type conversion

### PR DESCRIPTION
**Pull request by lrasmus**
Two tests for medications & procedures, based of @psbrandt's example

### CODE REVIEW

File: src/test/resources/phenotype/phenotype.omop.json [hunk 1]
```diff
@@ -2,6 +2,6 @@
...
...cluded\":false,\"includeDescendants\":false,\"includeMapped\":false}]}}],\"QualifiedLimit\":{\"Type\":\"First\"},\"ExpressionLimit\":{\"Type\":\"First\"},\"InclusionRules\":[{\"name\":\"PhenotypeCase\",\"description\":\"PhenotypeCase\",\"expression\":{\"Type\":\"ALL\",\"CriteriaList\":[{\"Criteria\":{\"ConditionOccurrence\":{\"CodesetId\":0,\"ConditionTypeExclude\":false}},\"StartWindow\":{\"Start\":{\"Coeff\":-1},\"End\":{\"Coeff\":1}},\"Occurrence\":{\"Type\":2,\"Count\":1,\"IsDistinct\":false},\"RestrictVisit\":false}],\"DemographicCriteriaList\":[],\"Groups\":[{\"Type\":\"ANY\",\"CriteriaList\":[{\"Criteria\":{\"ConditionOccurrence\":{\"CodesetId\":1,\"ConditionTypeExclude\":false}},\"StartWindow\":{\"Start\":{\"Coeff\":-1},\"End\":{\"Coeff\":1}},\"Occurrence\":{\"Type\":2,\"Count\":1,\"IsDistinct\":false},\"RestrictVisit\":false},{\"Criteria\":{\"ConditionOccurrence\":{\"CodesetId\":2,\"ConditionTypeExclude\":false}},\"StartWindow\":{\"Start\":{\"Coeff\":-1},\"End\":{\"Coeff\":1}},\"Occurrence\":{\"Type\":2,\"Count\":1,\"IsDistinct\":false},\"RestrictVisit\":false}],\"DemographicCriteriaList\":[],\"Groups\":[]}]}}],\"CollapseSettings\":{\"CollapseType\":\"ERA\",\"EraPad\":0}}"
```
Comment (psbrandt) [inline src/test/resources/phenotype/phenotype.omop.json]: Why did this file change?

**Comment by psbrandt** [comment]
This looks great! Just had the one comment, but happy to merge, since it looks like the tests passed.

File: src/test/resources/phenotype/phenotype.omop.json (hunk 1 above)
Comment (psbrandt) [reply src/test/resources/phenotype/phenotype.omop.json]: Okay, I guess it's because the new valuesets.

Comment (lrasmus) [reply src/test/resources/phenotype/phenotype.omop.json]: I integrated changes into the large testing CQL file, which introduced some new value sets.  They started appearing in this output.

**Pull request merged**